tests/test-mongoc-collection.c

```diff
@@ -1451,7 +1451,7 @@ test_aggregate_legacy (void *data)
 
    /* no "cursor" argument */
    request = mock_server_receives_command (
-      server, "db", MONGOC_QUERY_NONE,
+      server, "db", MONGOC_QUERY_SLAVE_OK,
```

Comment:
> Do we need this change?

The test does fail without this change. The only other use of mock_server_receives_command() in this file (excluding the other aggregate test below) was for an insert command, which we've decided not to inject slaveOk into per earlier discussion.